### PR TITLE
[Rhosinfra 42] Add unittest for no configuration file

### PR DESCRIPTION
**Pull request by aopincar**

### CODE REVIEW

**Comment by yfried-redhat** [comment]
retest

**Comment by yfried-redhat** [comment]
retest

**Comment by yfried-redhat** [comment]
@aopincar please make sure it passes the gate

**Comment by gamado** [comment]
test this please

**Comment by rhosqeauto** [comment]
Jenkins connectivity check, please ignore.

**Comment by gamado** [comment]
retest

**Comment by gamado** [comment]
retest this please

**Comment by gamado** [comment]
retest

File: cli/conf.py [hunk 1]
```diff
@@ -29,13 +37,17 @@ def load_config_file():
     for path in (env_path, cwd_path, utils.USER_PATH, utils.SYSTEM_PATH):
         if path is not None and os.path.exists(path):
             _config.read(path)
-            return _config
+            break
+    else:
+        LOG.warning("Configuration file not found, using InfraRed project dir")
+        from os.path import dirname
```
Comment (yfried-redhat) [inline cli/conf.py]: remove this line please. `os` is already imported.

**Pull request merged**